react/features/base/tracks/middleware.web.ts

```diff
@@ -142,6 +142,15 @@ MiddlewareRegistry.register(store => next => action => {
 
         if (typeof action.track?.muted !== 'undefined' && participantID && !local) {
             logTracksForParticipant(store.getState()['features/base/tracks'], participantID, 'Track updated');
+
+            // Notify external API when remote participant mutes themselves
```

Comment:
We also need to know when they unmute. Can't we do both in the same palce? It feels very odd to have the logic split in 2 places.